app/authentication.html

```diff
@@ -24,36 +27,31 @@
   <body class="skin-organic">
   <section role="region" class="vbox fit">
     <header>
-      <button id="close" data-icon="close" data-action="back">
+      <a href="question_list_helpee.html" id="close" data-icon="close" data-action="back">
```

Comment:
Can you make that `index.html`? It's not use in practice but it's useful in development.